config/legales/GB.yml

```
@@ -8,7 +8,7 @@ section_2: "Subject to Section 3 and 4 below, You hereby grant to OHM a worldwid
 section_3: "OHM agrees that it may only redistribute Your Contents under the terms defined in the Contents, as explicitly tagged at the object level. Any Contents not tagged with a license at the object level will be considered to be in the public domain (https://creativecommons.org/share-your-work/public-domain/cc0/)."
 active_defn_1: "An 'active contributor' is defined as:"
 active_defn_2: "a natural person (whether using a single or multiple accounts) who has edited the Project in any three calendar months from the last 12 months (i.e. there is a demonstrated interest over time); and has maintained a valid email address in their registration profile and responds  to a request to vote within 3 weeks."
-section_4: "At Your or the copyright owner’s option, OHM agrees to attribute You or the copyright owner. A mechanism will be provided, currently a web page <a href=\"OHM_Contributors\">https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/Contributing</a>."
+section_4: "At Your or the copyright owner’s option, OHM agrees to attribute You or the copyright owner. A mechanism will be provided, currently a web page <a href="https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/OHM_Contributors">https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/OHM_Contributors</a>."
```

Should the quotation marks be escaped?

```suggestion
section_4: "At Your or the copyright owner’s option, OHM agrees to attribute You or the copyright owner. A mechanism will be provided, currently a web page: <a href=\"https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/OHM_Contributors\">https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/OHM_Contributors</a>."
```